drivers/bridge/setup_ip_tables.go

```diff
@@ -10,21 +10,22 @@ import (
 
 // DockerChain: DOCKER iptable chain name
 const (
-	DockerChain    = "DOCKER"
-	IsolationChain = "DOCKER-ISOLATION"
+	DockerChain     = "DOCKER"
+	IsolationChain  = "DOCKER-ISOLATION"
+	IsolationChain2 = "DOCKER-ISOLATION-REV"
```

Comment:
Please rename `DOCKER-ISOLATION-REV` chain name and `IsolationChain2` variable to something more meaningful. Right now no one will understand what they are for.